src/nz/price-info.ts

```diff
@@ -22,7 +22,8 @@ import { Config } from './config';
     }
```

Comment:
Hi Frank,
Can you update nz-test-harness.html to have an example of how your new query works? other than that everything looks good

eg. :<script id='oxipay-price-info' src="dist/nz/Content/scripts/price-info.js?productPrice=500&noLogo=true"></script>